plugin/server_audit/server_audit.c

```diff
@@ -1398,7 +1398,7 @@ static size_t log_header(char *message, size_t message_len,
   if (output_type == OUTPUT_SYSLOG)
     return my_snprintf(message, message_len,
         "%.*s,%.*s,%.*s,%d,%lld,%s",
-        (unsigned int) serverhost_len, serverhost,
+        (int) serverhost_len, serverhost,
```

Comment:
why are we casting to int  - a size_t is `%zu`

Reply:
Because per [the C/C++ standard](https://en.cppreference.com/w/c/io/fprintf#Parameters), the `*`s in format specifiers take `signed int` rather than `unsigned int` or `size_t`… ¯\\\_🤨_/¯

Reply:
oops. Yes of course.

Reply:
🤔 The `my_snprintf` family is our own variant of `snprintf`, after all.
Is it worth to take `size_t` from `*`s, or is breaking from the Standard unjust?